Formula/cp2k.rb

```diff
@@ -23,20 +22,14 @@ class Cp2k < Formula
   fails_with :clang # needs OpenMP support
 
   resource "libint" do
-    url "https://downloads.sourceforge.net/project/libint/v1-releases/libint-1.1.5.tar.gz"
-    sha256 "31d7dd553c7b1a773863fcddc15ba9358bdcc58f5962c9fcee1cd24f309c4198"
-  end
-
-  # Upstream fix for GCC 10, remove in next version
-  # https://github.com/cp2k/dbcsr/commit/fe71e6fe
-  patch do
-    url "https://github.com/Homebrew/formula-patches/raw/0c086813/cp2k/gcc10.diff"
-    sha256 "dfaa319c999d49faae86cafe58ddb3b696f72a89f7cc85acd47b3288c6b9ac89"
+    url "https://github.com/cp2k/libint-cp2k/releases/download/v2.6.0/libint-v2.6.0-cp2k-lmax-4.tgz"
```

Comment:
Let's try to use `https://github.com/cp2k/libint-cp2k/releases/download/v2.7.0-beta.5/libint-v2.7.0-beta.5-cp2k-lmax-4.tgz` instead. I know it says beta, but then that's what they recommend to use along with the latest cp2k version…